src/main.ts

```diff
@@ -49,10 +49,14 @@ const run = async () => {
         - voulme mount .azure session token file between host and container,
         - volume mount temp directory between host and container, inline script file is created in temp directory
         */
+        let relative_path = process.env.GITHUB_ENV.substr(TEMP_DIRECTORY.length);

```

Comment:
rather than handling path as strings, use [Path](https://nodejs.org/api/path.html) module to find relative path and to join paths. This ensures that OS related differences in oath do no break things.